tasks/main.yml

```diff
@@ -130,6 +130,20 @@
       fail_on_error: yes
       strict: yes
 
+- name: Create/update templates in Kubernetes
```

Comment:
I believe this could lead to a scenario where new web pods come online with a new image (that expect new migrations to already have been run) and cause 500 errors. And then if the migrations fail, the deploy would be left in a weird state.

While not perfect, the existing order of operations where the migrations run first (and fail the deploy if they're bad) is a safer approach, even if there's a small window where the old pods are running on a migrated database. 

So I think this block should remain where it was.

Maybe we could extract the [Secret object creation](https://github.com/caktus/ansible-role-django-k8s/blob/v0.0.9/templates/web.yaml.j2#L7-L16) from `web.yaml` and just run that before the migration tasks?

Reply:
Agree that web pods should not be updated before migrations run. Agree that secrets maybe should be updated before migrations.

In the event the migrations fail, existing web pods will still be using the old secrets, but if one of those pods dies and is recreated it will start using the new secrets. Not sure if that's desired or not.

If it helps make things more predictable at least, the migration pods could theoretically use their own dedicated secrets.